great_expectations/expectations/expectation.py

```diff
@@ -1552,14 +1559,14 @@ def _get_renderer_diagnostics(
         expectation_type: str,
         test_diagnostics: List[ExpectationTestDiagnostics],
         registered_renderers: List[str],
-        standard_renderers: List[str] = [
-            "renderer.answer",
-            "renderer.diagnostic.unexpected_statement",
-            "renderer.diagnostic.observed_value",
-            "renderer.diagnostic.status_icon",
-            "renderer.diagnostic.unexpected_table",
-            "renderer.prescriptive",
-            "renderer.question",
+        standard_renderers: List[Enum] = [
```

Comment:
Could we define a `Literal` type for these.
https://mypy.readthedocs.io/en/stable/literal_types.html#literal-types-and-enums